packages/manager/tools/sao-ovh-manager-module/template/src/dashboard/index.js

```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+<% const pascalcasedName = this.camelcase(name, { pascalCase: true }) -%>
```

Comment:
Variable definition cannot be done at first?

Reply:
I find it hard to read with the import

Reply:
@antleblanc Moved variable declarations to the first line but kept the import separated